main.py

```diff
@@ -49,6 +49,194 @@
+    -`/last` shows last cat caught in channel you used the command in
+    -`/leaderboards` shows whos a sweat and whos not
+    -`/pig` the most engaging boring game
+    -`/ping` shows cat bots delay speed
+    """,
+
+    """⚙ **Commands List 7**
+
+    -`/prism` create a prism once you get 1 of each cat
+    -`/random` get a random cat
+    -`/rate` get a super accurate rating of something
+    -`/remind` get a reminder in the future (+-5 minutes)
+    -`/roll` roll or die roll or die roll or...
+    """,
+
+    """⚙ **Commands List 8**
+
+    -`/rps` ROCK,PAPER,SCISSORS
+    -`/slots` gamble your lifes savings of cats
+    -`/store` buy some rain wont yah
+    -`/tictactoe` pretty self-explanatory
+    -`/tiktok` Read text as Tiktok's TTS women
+    """,
+
+    """⚙ **Commands List 9**
+
+    -`/trade` the good ol' trading system
+    -`/vote` YOU SHOULD VOTE FOR CAT BOT NOW!
+    -`/Wiki` Visit the Offical Cat Bot Wiki
+    """,
+   
+
+    """🐈 **Meet The Cats (Fine - Rare)**
+        
+    - Fine Cat <:finecat:1253369190312706208>: The most common
+    - Nice Cat <:nicecat:1253369188043587594>: He's pretty nice i guess
+    - Good Cat <:goodcat:1253369185422282863>: Such a Good boy (Whoops wrong animal)
+    - Rare Cat <:rarecat:1253369888869974036>: He's Rare
+    """,
+
+    """ **Wild - Sus**
+    
+    - Wild Cat <:wildcat:1253369172847755284>: He's a wild one WATCH OUT
+    - Baby Cat <:babycat:1253369170121199677>: He's only a baby who knows maybe he'll be the next billionare
+    - Epic Cat <:epiccat:1253369167881441338>: Totally so EPIC
+    - Sus Cat <:suscat:1253369181018263714>: I wouldn't trust him...
+    """,
+
+    """ **Brave - Superior**
+    
+    - Brave Cat <:bravecat:1253369177411026964>: Such a Heroic Cat across the Land of Cats!
+    - Rickroll Cat <:rickrollcat:1253369165348081844>: YOU JUST GOT RICk ROLLED HAHAHA!
+    - Reverse Cat <:reversecat:1253369161766277250>: Reverse in talking im
+    - Superior Cat <:superiorcat:1253369159228719135>: He's Superior in his head
+    """,
+
+    """ **Trash - Corrupt**
+    
+    - Trash Cat <:trashcat:1253369886550524055>: He already knows dont keep him 
+    - Legendary Cat <:legendarycat:1253369884189003826>: LEGENDARY
+    - Mythic Cat <:mythiccat:1253369163196661862>: Mythical Such a Soothing word to say
+    - 8bit Cat <:8bitcat:1253369157525966909>: Retro Cat from the 80's coming through!
+    - Corrupt Cat <:corruptcat:1253369155835396136>: He drank expired milk thats why he's like this    
+    """,
+
+    """ **Professor - eGirl** 
+    
+    - Professor Cat <:professorcat:1253369153373343788>: Probably the wisest of the bunch
+    - Divine Cat <:divinecat:1253369151083249664>: God Like
+    - Real Cat <:realcat:1253369148529049735>: Yup, just your ordinary fellow cat
+    - Ultimate Cat <:ultimatecat:1253369146624966718>: Thats an ULTIMATE achievement if you ever get him
+    - eGirl Cat <:egirlcat:1253369144636870796>: Such a Baddie
+    """,
```

Comment:
showing all the cats is counterproductive to our goal of sort of trying to keep them hidden
commands list is useless just press /